[13.0] python libs versions for buster

Encountered following error while using freezegun on 13.0:
```
    from freezegun import freeze_time
usr/local/lib/python3.7/dist-packages/freezegun/__init__.py:9: in <module>
    from .api import freeze_time
usr/local/lib/python3.7/dist-packages/freezegun/api.py:26: in <module>
    _EPOCHTZ = datetime.datetime(1970, 1, 1, tzinfo=dateutil.tz.UTC)
E   AttributeError: module 'dateutil.tz' has no attribute 'UTC'
```
This error is fixed by bumping `python-dateutil` lib to `2.7.3` as per odoo requirements.txt: https://github.com/odoo/odoo/blob/13.0/requirements.txt

However, it seems there are quite a few more libs to update according to following commit on odoo: https://github.com/odoo/odoo/commit/32e455bf72980e6330871aa9cd99c26c6e1225d7

--- a/13.0/base_requirements.txt
+++ b/13.0/base_requirements.txt
@@ -1,56 +1,55 @@
 # Odoo dependencies
-Babel==2.3.4
+Babel==2.6.0
 chardet==3.0.4
-decorator==4.0.10
-docutils==0.12
+decorator==4.3.0
+docutils==0.14
 ebaysdk==2.1.5
 feedparser==5.2.1
-gevent==1.3.4 ; sys_platform != 'win32' and python_version >= '3.7'
-greenlet==0.4.13 ; python_version >= '3.7'
-html2text==2016.9.19
+gevent==1.3.7 ; sys_platform != 'win32' and python_version >= '3.7'
+greenlet==0.4.15 ; python_version >= '3.7'
+html2text==2018.1.9
 Jinja2==2.10.1
-libsass==0.12.3
-lxml==4.2.3 ; sys_platform != 'win32' and python_version >= '3.7'
-Mako==1.0.4
-MarkupSafe==0.23
+libsass==0.17.0
+lxml==4.3.2 ; sys_platform != 'win32' and python_version >= '3.7'
+Mako==1.0.7
+MarkupSafe==1.1.1
 mock==2.0.0
-num2words==0.5.6
-ofxparse==0.16
-passlib==1.6.5
+num2words==0.5.10
+ofxparse==0.19
+passlib==1.7.2
 Pillow==6.2.2
 polib==1.1.0
 psutil==5.7.0; sys_platform != 'win32'
-psycopg2==2.7.3.1; sys_platform != 'win32'
-pydot==1.2.3
-pyldap==2.4.28; sys_platform != 'win32'
-pyparsing==2.1.10
+psycopg2==2.7.7; sys_platform != 'win32'
+pydot==1.4.1
+python-ldap==3.2.0; sys_platform != 'win32'
+pyparsing==2.2.0
 PyPDF2==1.26.0
-pyserial==3.1.1
-python-dateutil==2.5.3
-pytz==2016.7
-pyusb==1.0.0
-qrcode==5.3
-reportlab==3.3.0
-requests==2.20.0
-zeep==3.1.0
+pyserial==3.4
+python-dateutil==2.7.3
+pytz==2020.1
+pyusb==1.0.2
+qrcode==6.1
+reportlab==3.5.42
+requests==2.23.0
+zeep==3.2.0
 vatnumber==1.2
-vobject==0.9.3
-Werkzeug==0.16.0
-XlsxWriter==0.9.3
+vobject==0.9.6.1
+Werkzeug==0.16.1
+XlsxWriter==1.1.2
 xlwt==1.3.*
-xlrd==1.0.0
+xlrd==1.1.0
 
 # Not part of official requirements, but used by some addons
-chardet==3.0.4
 # colorama==0.3.9
 gdata==2.0.18
 html5lib==1.0.1
-odfpy==1.3.5
+odfpy==1.4.1
 odoo-autodiscover==2.0.0
 pyinotify==0.9.6
-python-stdnum==1.9
-simplejson==3.11.1
-urllib3==1.24.2
+python-stdnum==1.13
+simplejson==3.17.0
+urllib3==1.25.9
 
 # Migration tools
 marabunta==0.10.4

--- a/HISTORY.rst
+++ b/HISTORY.rst
@@ -35,6 +35,7 @@ Unreleased
 
 * Bump `psutil` version to 5.7.0
 * Bump `Pillow` version to 6.2.2 (v9-13)
+* Bump various libraries in v13 to match Odoo's requirements
 
 **Build**
 

